data/governments.txt

```diff
@@ -8,6 +8,20 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+"reputation titles"
+	"-infinity" "'You. Will. Die.'"
+	-100000 "Hated Adversary"
+	-10000 "Feared Enemy"
+	-1000 "Hostile Relations"
+	-100 "Wanted Person"
+	-10 "Petty Disputes"
+	0 "Civil Interactions"
```

Comment:
Just a thought, but we could have an "unknown" one by default? (not that exact title but the idea of it)

I think that around this they just don't know who you are, and this way it doesn't cause any incoherence if you haven't actually met the faction.